### DISCUSSION TITLE
problems with AuthMe and online accounts

### DISCUSSION THREAD

**Discussion by FallenCrystal** [games647/FastLogin#901] [Q&A] [answered]
i used online auth for me but i still need to enter my password

**Comment by TuxCoding** [comment, marked as answer]
Please post your server log and further details about your setup. You can view a template for bug reports here: https://github.com/games647/FastLogin/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml

**Reply by FallenCrystal** [reply]
opened issue #902
